recipes/vowpalwabbit/meta.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: vowpalwabbit
+  version: 8.7.0
+
+source:
+  url: https://scgrahamarchive.blob.core.windows.net/vowpal-wabbit/vowpal_wabbit_2ee2bcf_rapidjson.tgz 
+  sha256: befed5ecd42decd533b8e11a98f0e76f9d48951c085b52786277d86ec06923ac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.13
+  host:
+    - py-boost
+    - pip
+    - python
+  run:
+    - py-boost
+    - python
+
+test:
+  imports:
+    - vowpalwabbit
+  requires:
+    - numpy
+    - pandas
+    - pytest
+    - scipy
+    - scikit-learn
+  source_files:
+    - python/tests
+  commands:
+    - py.test python/tests
+
+about:
+  home: https://github.com/VowpalWabbit/vowpal_wabbit
+  license: BSD
```

Which BSD licence is it? From a quick glance it looks like this but please check:
```suggestion
  license: BSD-3-Clause
```